app/css/interceptor.css

```diff
@@ -499,6 +499,31 @@ svg.spinner > circle {
 	grid-template-columns: max-content auto auto auto auto;
 	width: 100%;
 }
+.log-table-9 {
+	display: grid;
+	grid-template-columns: max-content auto auto auto auto auto auto auto auto;
+	width: 100%;
+}
```

Comment:
i believe this can be achieved in a very similar manner as a single class declaration given the grid only consist of a single row.

```css
.log-table {
	display: grid;
	grid-template-columns: max-content;
	grid-auto-columns: auto;
	grid-auto-flow: column;
	width: 100%;
}
```